crates/bitwarden-fido/src/authenticator.rs

```diff
@@ -229,12 +229,8 @@ impl<'a> Fido2Authenticator<'a> {
         };
 
         let authenticator_data = response.auth_data.to_vec();
-        let credential_id = response
-            .auth_data
-            .attested_credential_data
-            .ok_or(GetAssertionError::MissingAttestedCredentialData)?
-            .credential_id()
-            .to_vec();
+        let credential_id =
+            string_to_guid_bytes(&self.get_selected_credential()?.credential.credential_id)?;
```

Comment:
We should probably extract the call to `get_selected_credential()` to a separate variable, so we don't have to call it twice (once here and another when setting `selected_credential` on line 244)

Reply:
Fixed!